UpdateExpiredCgn/table.ts

```diff
@@ -10,24 +12,33 @@ import {
   TableEntry
 } from "../utils/table_storage";
 
+const ExpiredCgnRowKeyType = t.interface({
```

Comment:
Maybe we don't need `-Type` suffix, as we already differentiate types with the initial capital letter
```suggestion
const ExpiredCgnRowKey = t.interface({
```